activerecord/Rakefile

```diff
@@ -109,20 +127,38 @@ namespace :db do
     desc "Build the PostgreSQL test databases"
     task :build do
       config = ARTest.config["connections"]["postgresql"]
-      %x( createdb -E UTF8 -T template0 #{config["arunit"]["database"]} )
-      %x( createdb -E UTF8 -T template0 #{config["arunit2"]["database"]} )
+
+      if ENV["TRAVIS"]
+        %x(createdb -E UTF8 -T template0 #{config["arunit"]["database"]})
+        %x(createdb -E UTF8 -T template0 #{config["arunit2"]["database"]})
+      else
+        puts "\nCreating database '#{config['arunit']['database']}' as PostgreSQL user '#{config['arunit']['username']}'"
+        %x(createdb -E UTF8 -T template0 --host=#{config["arunit"]["host"]} --username=#{config["arunit"]["username"]} --password #{config["arunit"]["database"]})
+
+        puts "\nCreating database '#{config['arunit2']['database']}' as PostgreSQL user '#{config['arunit2']['username']}'"
+        %x(createdb -E UTF8 -T template0 --host=#{config["arunit2"]["host"]} --username=#{config["arunit2"]["username"]} --password #{config["arunit2"]["database"]})
```

Comment:
Not sure why the password option is used here, not everyone uses password based config, for example pg on osx. This is not going to work in that case.